pkg/providers/instancetype/suite_test.go

```diff
@@ -2552,11 +2559,13 @@ var _ = Describe("InstanceTypeProvider", func() {
 })
 
 func uniqueInstanceTypeList(instanceTypesLists [][]*corecloudprovider.InstanceType) {
+	GinkgoHelper()
 	for x := range instanceTypesLists {
 		for y := range instanceTypesLists {
 			if x == y {
 				continue
 			}
+			fmt.Printf("i:%d, j:%d\n", x, y)
```

Comment:
What's this for?

Reply:
Missed dropping a couple of debug prints, this included. Dropped now.